PublicationsRouter/v2/api/API.md

```diff
@@ -74,76 +74,94 @@ If you have publicly hosted content (e.g. splash pages, full-text web pages, or
+    Body: 
+        {Incoming Notification JSON}
     
-    [Incoming Notification JSON]
+### 2. Validate List of notifications with Metadata-only request
+
+If you are sending a list of notifications, the request must take the form:
+
+    POST /validate/list?api_key=<api_key>
+    Header:
+        Content-Type: application/json
+    Body:
+        # List of Incoming Notification JSON
+        [{"notification": Incoming, "id": 1}, {"notification": Incoming, "id": 2}, {"notification": Incoming, "id": 3}...]    
 
-#### Metadata + Package request
+NOTE: Make sure that an ID is sent for each Incoming notification as those IDs will be returned into a success or error list 
+
+### 3. Validate Metadata + Package request
 
 If you are sending binary content as well as the metadata, the request must take the form:
 
     POST /validate?api_key=<api_key>
-    Content-Type: multipart/form-data; boundary=FulltextBoundary
-    
-    --FulltextBoundary
-    
-    Content-Disposition: form-data; name="metadata"
-    Content-Type: application/json
-    
-    [Incoming Notification JSON]
-    
-    --FulltextBoundary
+    Header:
+        Content-Type: multipart/form-data; boundary=FulltextBoundary
     
-    Content-Disposition: form-data; name="content"
-    Content-Type: application/zip
-    
-    [Package]
-    
-    --FulltextBoundary--
+    Body:
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="metadata"
+        Content-Type: application/json
+
+        [Incoming Notification JSON]
+
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="content"
+        Content-Type: application/zip
+
+        [Package]
 
-If you are carrying out this request you MUST include the **content.packaging_format** field in the notification metadata and populate it with the appropriate format identifier as per the [Packaging Format](https://github.com/sherpaservices/Public-Documentation/blob/PublicationsRouter/api/v2/Packaging.md) documentation.
+        --FulltextBoundary--
 
-#### Minimum Metadata + Package request
+If you are carrying out this request you MUST include the **content.packaging_format** field in the notification metadata and populate it with the appropriate format identifier as per the [Packaging Format](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md) documentation.
+
+### 4. Validate Minimum Metadata + Package request
 
 It is possible to send a request with virtually no JSON metadata, instead relying on metadata embedded in the binary Package (e.g. in a JATS XML structure).
 
-To do this, send the bare-minimum JSON notification, with only the format identifier of the package included.  For example:
+To do this, send the bare-minimum JSON notification, with only the format identifier of the [package](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md) included.  For example:
```

Comment:
I wonder if we should change the preceding line from:
`It is possible to send a request with virtually no JSON metadata, instead relying on metadata embedded in the binary Package (e.g. in a JATS XML structure).`
to:
`It is possible to send a request with virtually no JSON metadata, instead relying on metadata embedded in an XML file in the binary Package (e.g. in a JATS XML structure).`